sql-statements/sql-statement-flashback-to-timestamp.md

```diff
@@ -0,0 +1,103 @@
+---
+title: FLASHBACK CLUSTER TO TIMESTAMP
+summary: Learn the usage of FLASHBACK CLUSTER TO TIMESTAMP in TiDB databases.
+---
+
+# FLASHBACK CLUSTER TO TIMESTAMP
+
+TiDB v6.4.0 introduces the `FLASHBACK CLUSTER TO TIMESTAMP` syntax. You can use it to restore a cluster to a specific point in time.
+
+> **Warning:**
+>
+> Before executing `FLASHBACK CLUSTER TO TIMESTAMP`, you need to pause PITR and TiCDC tasks and restart them after the `FLASHBACK` is completed. Otherwise, it might lead to synchronization failure.
```

Comment:
```suggestion
> Before executing `FLASHBACK CLUSTER TO TIMESTAMP`, you need to pause PITR and TiCDC tasks and restart them after the `FLASHBACK` is completed. Otherwise, it might lead to replication failure.
```